python/py-svgelements/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-svgelements
+version             1.6.10
+revision            0
+
+license             MIT
+maintainers         @abey79 openmaintainer
+supported_archs     noarch
+
+description         SVG Parsing for Elements, Paths, and other SVG Objects.
+long_description    svgelements does high fidelity SVG parsing and geometric rendering. \
+                    The goal is to successfully and correctly process SVG for use with \
+                    any scripts that may need or want to use SVG files as geometric data.
+
+homepage            https://github.com/meerk40t/svgelements
+
+checksums           rmd160  ad0e9628f9f8c233ae07c127a7746973c23ef14d \
+                    sha256  80ca705b26103a46256bdbc80ce75e4aa174d94fa7f57a5c9b9253208cd3b6f3 \
+                    size    143802
+
+python.versions     36 37 38 39 310
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    livecheck.type  none
```

Comment:
if you do not have multiple `version` fields in the Portfile (most common case), the the `python` PG will set this automatically so you can remove the `livecheck.type` field here and in the other ones